recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -21,6 +21,11 @@ if "%CONDA_BLD_PATH%" == "" (
     set "CONDA_BLD_PATH=C:\\bld\\"
 )
 
+:: Increase pagesize, cf. https://github.com/conda-forge/conda-forge-ci-setup-feedstock/issues/155
+if "%CONDA_BLD_PATH%" == "C:\\bld\\" (
```

Comment:
can you also check that `%CI%` is azure?

Reply:
I'm assuming this takes the value `AZURE`? Feel free to correct if wrong, I need to log off now.

Reply:
It should be all lowercase